src/sentry/templates/sentry/emails/_group.html

```diff
@@ -1,12 +1,15 @@
 {% load sentry_helpers %}
-
-{% with type=group.get_event_type metadata=group.get_event_metadata transaction=group.culprit %}
+{% with type=group.get_event_type metadata=group.get_event_metadata transaction=group.culprit extra_params=link_params_for_rule %}
 <div class="issue {% if group.is_resolved %}resolved{% endif %}">
   {% if type == "error" %}
     <div class="event-type error">
       <h3>
         {% if metadata.type %}
-          <a href="{{group.get_absolute_url}}">{{metadata.type|truncatechars:40}}</a>
+          {%if rule %}
+            <a href="{{group.get_absolute_url}}{{extra_params|get_item:rule.id}}">{{metadata.type|truncatechars:40}}</a>
+          {% else %}
+            <a href="{{group.get_absolute_url}}">{{metadata.type|truncatechars:40}}</a>
+          {% endif %}
```

Comment:
Wondering if it makes sense to do something like this:
```
          {%if rule %}
            {% url_params = extra_params|get_item:rule.id %}
          {% else %}
            {% url_params = "" %}
          {% endif %}
```
(this is just pseudocode, I don't know what the syntax for this would look like off the top of my head)
and then use `url_params` on each url, to avoid having this else statement in multiple places.

Reply:
I tried this before using `with` to declare vars but it doesn't look legal:
`
{% if rule %}
    {% with url_params=extra_params|get_item:rule.id %}
{% else %}
  {% with url_params="" %}
{% endif %}
<!-- more templating here ...>


{% endwith %}
`

I doesn't look like django templating allows starting a new templatetag before closing out `with` tags.